deployment/roles/haystack/tasks/main.yml

```diff
@@ -77,3 +77,10 @@
   community.docker.docker_compose:
     project_src: "{{ client_installation_directory }}/"
     build: yes
+- name: "{{ client }} | change group ownership (https://github.com/ansible/ansible/issues/51601)"
```

Comment:
maybe it's a stupid question, but why is this line necessary since we already have a group ownership defined line 27 ?

Reply:
Not stupid at all, this is an issue in ansible, related to the git module: https://github.com/ansible/ansible/issues/51601

Reply:
so we need to specify it twice. ok